test/farm/001-farm.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+#
+# Tests of podman farm commands
+#
+
+load helpers.bash
+
+###############################################################################
+# BEGIN tests
+
+fname="test-farm"
+containerfile="test/farm/Containerfile"
+
+@test "farm - check farm has been created" {
+    run_podman farm ls
+    assert "$status" -eq 0 "get list of farms"
+    assert "$output" =~ $fname
+    assert "$output" =~ "test-node"
+}
+
+@test "farm - build on local only" {
+    iname="test-image-1"
+    empty_farm="empty-farm"
+    # create an empty farm
+    run_podman farm create $empty_farm
+    assert "$status" -eq 0 "create empty farm"
+    run_podman farm --farm $empty_farm build -f $containerfile -t $iname .
+    assert "$status" -eq 0 "create new manifest list $iname"
+    assert "$output" =~ "Local builder ready"
+    # inspect manifest list built and saved in dir
+    run_podman manifest inspect $iname
+    assert "$status" -eq 0 "inspect manifest list $iname"
+    assert "$output" =~ "amd64"
+    # cleanup images on node
+    ssh $ROOTLESS_USER@localhost podman rmi -af
+    assert "$status" -eq 0 "remove built image on remote node"
+}
+
+@test "farm - build on farm node and local" {
+    iname="test-image-2"
+    run_podman farm build -f $containerfile -t $iname .
+    assert "$status" -eq 0 "create new manifest list $iname"
+    assert "$output" =~ "Farm \"$fname\" ready"
+    # inspect manifest list built and saved in dir
+    run_podman manifest inspect $iname
+    assert "$status" -eq 0 "inspect manifest list $iname"
+    assert "$output" =~ "amd64"
+    # cleanup images on node
+    ssh $ROOTLESS_USER@localhost podman rmi -af
+    assert "$status" -eq 0 "remove built image on remote node"
+}
+
+@test "farm - build on farm node only with --cleanup" {
+    iname="test-image-3"
+    run_podman farm build -f $containerfile --cleanup --local=false -t $iname .
+    assert "$status" -eq 0 "create new manifest list $iname"
+    assert "$output" =~ "Farm \"$fname\" ready"
+    # inspect manifest list built and saved in dir
+    cat $iname/manifest.json
```

Comment:
Debug artifact can be removed.

Reply:
That is part of the test, when we build from remote nodes only the manifest list is saved in a dir, so checking the json for the manifest list there to ensure it includes the architecture we are expecting.

Reply:
I think you need to add `run cat` to inspect the output.  Otherwise the output of line 55 is checked.

Reply:
fixed